### PR TITLE
Support helm repo for cloudcore

### LINKED ISSUE #3300: Plan for containerized deployment for cloudcore on k8s platform.
Plans:
- [x] Support helm repo for KubeEdge cloudcore. https://github.com/kubeedge/kubeedge/pull/3304
- [ ] Helm CI test.  #3418 
- [ ] Custom keadm build for the edgecore. https://github.com/kubeedge/kubeedge/pull/3320
- [x] External iptables-manager component integration, considers a complete externality later and needs to optimize the code.  https://github.com/kubeedge/kubeedge/pull/3265

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <zhu733756@kubesphere.io>

**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:
Helm repo support for the cloudside, make it more easier to deploy applications on the k8s platform.

**Which issue(s) this PR fixes**:

Fixes #3300 

**Special notes for your reviewer**:

Only support non-ha deployment for cloudcore.

**Does this PR introduce a user-facing change?**:

```release-note
Support helm repo for the cloudcore.
```
/cc @fisherxu

### CODE REVIEW

File: build/helm/cloudcore/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: cloudcore
+version: 0.1.0
+appVersion: 1.8.0
```
Comment (gy95) [inline build/helm/cloudcore/Chart.yaml:4]: 1.8.2?

File: build/helm/cloudcore/crds/cluster_objectsync_v1alpha1.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: clusterobjectsyncs.reliablesyncs.kubeedge.io
```
Comment (gy95) [inline build/helm/cloudcore/crds/cluster_objectsync_v1alpha1.yaml:9]: what's difference of these defined files and files in https://github.com/kubeedge/kubeedge/tree/master/build/crds/devices

Comment (zhu733756) [reply build/helm/cloudcore/crds/cluster_objectsync_v1alpha1.yaml:9]: No different, just copy it here to install them with helm.

File: build/helm/cloudcore/Chart.yaml (hunk 1 above)
Comment (zhu733756) [reply build/helm/cloudcore/Chart.yaml:4]: Agree with you, will be fixed.

**Comment by zhu733756** [comment]
The edgecore can be deployed by custom keadm, see https://github.com/kubeedge/kubeedge/pull/3320

**Comment by fisherxu** [comment]
We can add some e2e for this later :) @zhu733756

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Comment by zhu733756** [comment]
> We can add some e2e for this later :) @zhu733756

OK

**Pull request merged**